modules/chargebee-integration/app/webhook.go

```diff
@@ -9,9 +9,13 @@ import (
 	"net/http"
 	"time"
 
+	"github.com/kuberlogic/kuberlogic/modules/dynamic-apiserver/pkg/generated/models"
 	"go.uber.org/zap"
+)
 
-	"github.com/kuberlogic/kuberlogic/modules/dynamic-apiserver/pkg/generated/models"
+const (
+	ChargebeePrefixCustomField = "cf_"
```

Comment:
As I understand it's never used in master.

Reply:
Done